SCClassLibrary/Platform/Platform.sc

```diff
@@ -91,9 +91,12 @@ Platform {
 		}
 	}
 
-	loadStartupFiles { this.startupFiles.do{|afile|
-		afile = afile.standardizePath;
-		if(File.exists(afile), {afile.load})
+	loadStartupFiles {
```

Comment:
Do you mean adding running a "startup.scd" file which is in the same directory as the config file ? And I suppose in that case the global "startup.scd" would not run. That would be easy to implement.